src/main/java/edu/harvard/hul/ois/fits/Fits.java

```diff
@@ -288,11 +294,14 @@ public static void main( String[] args ) throws FitsException, IOException, Pars
     } else {
       traverseDirs = false;
     }
-    if (cmd.hasOption( "n") ) {
+    if (cmd.hasOption( "n" ) ) {
       nestDirs = true;
     } else {
       nestDirs = false;
     }
+    if (cmd.hasOption( "t" )) {
```

Comment:
This could be:

```java
rawToolOutput = cmd.hasOption("t");
```

Reply:
Sure, was just following other command line options but this is definitely better.